packages/v4/patternfly-docs/content/accessibility/brand/brand.md

```diff
@@ -3,14 +3,51 @@ id: Brand
 section: components
 ---
 
-A **brand** is used to place a product logotype on a screen.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Keyboard users** should not be able to focus on the brand, as it is not interactive.
+## Accessibility
 
-**Screen reader users** should be able to navigate to the brand and have a screen reader read the brand’s descriptive text.
+To implement an accessible PatternFly **brand** component:
 
-The following props/attributes have been added for you or are customizable in PatternFly:
+- Do not make the brand itself interactive or focusable via keyboard
+- Apply alternative text to the brand
+- Ensure assistive technologies such as screen readers can navigate to the brand if there is alternative text applied
 
-| React prop | React component it should be applied to | Which HTML element it appears on in markup | Reason used |
-| -- | -- | -- | -- |
-| alt | Brand | `.pf-c-brand` | Descriptive alternative text for the image of the Brand |
\ No newline at end of file
+## Testing
+
+At a minimum, a brand should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="brand-a11y-checkbox-1" label="The brand itself is not interactive and cannot be focused via keyboard." description="If a brand is placed inside of an anchor element, the anchor element should still receive focus." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="brand-a11y-checkbox-2" label={<span>The brand has alternative text via <code className="ws-code">alt</code>.</span>} description={<span>If the brand is only decorative, the <code className="ws-code">alt</code> should have an empty string passed in as a value.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="brand-a11y-checkbox-3" label="The alternative text of the brand is announced to users via assistive technologies when they navigate to it." description="If the brand is decorative and is given an empty string for its alternative text, the brand should instead be skipped by assistive technologies." />
+  </ListItem>
+</List>
+
+## React customization
+
+The following React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason |
+| -- | -- | -- |
+| `alt="[the brand name or text describing the brand]"` | `Brand` | Provides an accessible description of the brand. **Required** <br/><br/> This prop will be used for all `source` elements and the fallback `img` when a `picture` brand is used. |
+
+## HTML/CSS customization
+
+The following HTML attributes and PatternFly classes can be used for more fine-tuned control over accessibility.
+
+| Attribute or class | Applied to | Reason | 
+|---|---|---|
+| `alt="[the brand name or text describing the brand]"` | `.pf-c-brand > img` | Provides an accessible description of the brand. **Required** <br/><br/> This prop will be used for all `source` elements and the fallback `img` when a `picture` brand is used. |
```

Comment:
Technically an attribute, not a prop :) if you feel like changing it!

Reply:
At least I didn't copy+paste the double m'd "minimum" again 😆 Fixed!